packages/app-builder/src/routes/_builder+/_layout.tsx

```diff
@@ -69,15 +70,21 @@ export async function loader({ request }: LoaderFunctionArgs) {
   }
 
   const firstSettings = getSettings(user)[0];
-  const [organizationDetail, orgUsers, orgTags, versions, datasetFreshnessInfo] = await Promise.all(
-    [
-      organization.getCurrentOrganization(),
-      organization.listUsers(),
-      organization.listTags(),
-      versionRepository.getBackendVersion(),
-      getDatasetFreshnessInfo(sanctionCheck, scenario),
-    ],
-  );
+  const [organizationDetail, orgUsers, orgTags, versions] = await Promise.all([
+    organization.getCurrentOrganization(),
+    organization.listUsers(),
+    organization.listTags(),
+    versionRepository.getBackendVersion(),
+  ]);
+
+  let datasetFreshnessInfo: { lastExport: string } | null = null;
+  try {
+    datasetFreshnessInfo = await getDatasetFreshnessInfo(sanctionCheck, scenario);
+  } catch (err) {
+    if (!isHttpError(err)) {
+      captureRemixServerException(err, 'remix.server', request, true);
+    }
+  }
```

Comment:
What do you think if we turn the `Promise.all()` into a `Promise.allSettled()` to keep the `getDatasetFreshnessInfo()` method parallelised and handle the errors gracefully ?